qa/automation/CareerPortal.Tests.Scenarios/Sataeva_cv form_feature/CV_form.feature

```diff
@@ -0,0 +1,181 @@
+Feature: Send CV form
+
+Background:
+Given user opened https://career.quantori.com/
+And user scrolled down to the "Send us your CV" form
+
+Scenario Outline: checking if the field "Name" is filled out with correct amount of symbols 
+When user entered text with <Amount> of symbols in "Name" field
+And "Name" field consists of 2 words
+And these two words are separated with one space
+Then the result should be <Result> on the screen
+
+Examples:
+  | Amount | Result                                                                                  | 
+  | 69     | no error message, line under the field is green                                         |
+  | 70     | no error message, line under the field is green                                         | 
+  | 71     | line under the field is red, error message "The maximum number of characters is 70"     | 
+
+
+Scenario: field "Name" consists of one word 
+When user types "Hello" in field "Name"
+Then user should get an error message "Invalid name format"
+And line under the field should turn to red
+
+Scenario: field "Name" consists of two words, separated with one space
+When user types "Hello world" in field "Name"
+Then line under the field should turn to green
+And user should not get any "Error" message
```

Comment:
It is not necessary to discribe an error case in positive scenario

Reply:
You are right. :)
Done.